tox.ini

```diff
@@ -48,6 +48,7 @@ commands =
 [testenv:docs,docs-py{38,39,310,311}]
 description = Build documentation
 extras = docs
+passenv = BAYBE_DOCS_LINKCHECK_IGNORE
```

Comment:
This line ensures that set linkcheck environment variables is just passed. Tested this on my fork, and it works.